test/Setup.t.sol

```diff
@@ -3,18 +3,18 @@ pragma solidity >=0.8.9;
 
 /// library imports
 import {Test, Vm} from "forge-std/Test.sol";
-import "forge-std/Console.sol";
+import "forge-std/console.sol";
 
 /// @dev imports from Pigeon Helper (Facilitate State Transfer Mocks)
 import {WormholeHelper} from "pigeon/wormhole/WormholeHelper.sol";
 import {AxelarHelper} from "pigeon/axelar/AxelarHelper.sol";
 
 /// local imports
-import {WormholeSenderAdapter} from "../src/adapters/wormhole/WormholeSenderAdapter.sol";
-import {WormholeReceiverAdapter} from "../src/adapters/wormhole/WormholeReceiverAdapter.sol";
+import {WormholeSenderAdapter} from "src/adapters/Wormhole/WormholeSenderAdapter.sol";
```

Comment:
Should we unify on the casing? I think folder names should be `wormhole` and `celer`.

Reply:
SG

Reply:
added this to the list in issue #43